recipes/opus/all/conanfile.py

```diff
@@ -56,6 +62,9 @@ def generate(self):
         tc = CMakeToolchain(self)
         tc.variables["OPUS_FIXED_POINT"] = self.options.fixed_point
         tc.variables["OPUS_STACK_PROTECTOR"] = self.options.stack_protector
+        tc.variables["OPUS_OSCE"] = self.options.osce
+        tc.variables["OPUS_DEEP_PLC"] = self.options.deep_plc
+        tc.variables["OPUS_DRED"] = self.options.dred
```

Comment:
```suggestion
        tc.variables["OPUS_DRED"] = self.options.dred
        tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
```
Needed for `tc.variables` to set CMake `options()` correctly.

Reply:
Could you please explain the need for this CMake policy @valgur ? 🤔

Reply:
Without it any values set using `option()` in CMake do not get overridden by `tc.variables[...]`. `tc.variables` sets its values using `set()` commands loaded within the `project()` command. `tc.cache_variables` get set via `-D...` command line arguments. Only the latter work correctly when this policy is not enabled.

The policy gets set automatically when CMake's policy version is set to 3.12 or above, which is usually set via `cmake_minimum_required()`.

Tbh, this should be set by Conan automatically. I consider it a bug.

More info in CMake docs: https://cmake.org/cmake/help/latest/policy/CMP0077.html